process/transaction/shardProcess.go

```diff
@@ -275,7 +275,12 @@ func (txProc *txProcessor) executingFailedTransaction(
 	return process.ErrFailedTransaction
 }
 
-func (txProc *txProcessor) createReceiptWithReturnedGas(txHash []byte, tx *transaction.Transaction, acntSnd state.UserAccountHandler) error {
+func (txProc *txProcessor) createReceiptWithReturnedGas(
```

Comment:
this will have some conflicts with my PR.

Reply:
will solve them later